logic/transaction.js

```diff
@@ -696,21 +696,21 @@ class Transaction {
 		}
 
 		const verifyTransactionTypes = (
-			transaction,
-			sender,
-			tx,
+			aTransaction,
+			aSender,
+			aTx,
 			verifyTransactionTypesCb
```

Comment:
Really not look good this naming convention. `aTransaction`, `aSender`. 

May if you need distinguish params by function name. ` verificationTransaction`, `verificationSender`